pkg/database/models/account.go

```diff
@@ -17,7 +17,7 @@ type Account struct {
 	LastName  string
 	Passwd    string // Considered as plaintext, but can be encrypted by frontend
 
-	Gender   bool
+	Gender   string
```

Comment:
Uhh, so what are the strings to fill? "female" or "Female" of simply "f"? name it.